src/parsers/parser_nodes.ts

```diff
@@ -1,14 +1,15 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+/* eslint-disable max-classes-per-file */
```

Comment:
In my opinion we could also get rid of the `max-classes-per-file` policy project-wide.

Reply:
Sure.  

What is your opinion of placement of the copyright notice vs linting configuration?  I had just left it where the intellisense put it, at the very top.